Adyen/Validators/BrazilSocialSecurityNumberValidator.swift

```diff
@@ -0,0 +1,71 @@
+//
+// Copyright (c) 2024 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+@_spi(AdyenInternal)
+public final class BrazilSocialSecurityNumberValidator: CombinedValidator, StatusValidator {
+    
+    private enum Constants {
+        static let firstMaxLength = 11
+        static let secondMaxLength = 14
+    }
+    
+    public let firstValidator: Validator
+    
+    public let secondValidator: Validator
+    
+    public init() {
+        self.firstValidator = NumericStringValidator(exactLength: Constants.firstMaxLength)
+        self.secondValidator = NumericStringValidator(exactLength: Constants.secondMaxLength)
+    }
+    
+    public func validate(_ value: String) -> Adyen.ValidationStatus {
```

Comment:
looks like autocompletion - the Adyen namespace should not be needed here right?
```suggestion
    public func validate(_ value: String) -> ValidationStatus {
```

Reply:
Good catch, updated!